terraform/install.sh

```diff
@@ -278,32 +290,72 @@ checkAuthentication() {
         log "Authentication failed"
         log "Please allow gcloud and Cloud Shell to access your GCP account"
     fi
-    while [[ -z $AUTH_ACCT  && "${TRIES}" -lt 10  ]]; do
+    while [[ -z $AUTH_ACCT  && "${TRIES}" -lt 30  ]]; do
         AUTH_ACCT=$(gcloud auth list --format="value(account)")
-        sleep 3;
+        sleep 1;
```

Comment:
This might be not enough time for something to change, so second retry can still fail with high probability if it's a transient error

Reply:
In this case, we're waiting for the user to accept the dialog box. Since we're just doing a poll and wait here, there's no real downside to checking often, and a quicker poll time makes the script respond faster after the user finishes their task. I didn't want the user staring at the `Please allow gcloud and Cloud Shell to access your GCP account` error message for 3 seconds wondering what they did wrong 

But I'm fine with reverting too, because a 3 second wait isn't much of an issue